api/src/main/java/io/cloudevents/rw/CloudEventDataMapper.java

```diff
@@ -37,4 +37,8 @@
      */
     R map(CloudEventData data) throws CloudEventRWException;
 
+    /**
+     * No-op mapper which can be used as default when no mapper is provided
+     */
+    CloudEventDataMapper<?> NOOP = d -> d;
```

Comment:
It might be better for this to be a static method, like [`java.util.function.Function.identity()`](https://github.com/openjdk/jdk/blob/jdk8-b120/jdk/src/share/classes/java/util/function/Function.java#L97) and perhaps also called `identity()`:

```java
  static <R extends CloudEventData> CloudEventDataMapper<R> identity() {
    return d -> d;
  }
```

The `<?>` looks like it might make `NOOP` less useful than it could be.

Reply:
Sure!

Reply:
I think it needs to be a wildcard... Otherwise it requires a cast of `d` to `R`

Reply:
Check it out now